src/index.jsx

```diff
@@ -13,6 +14,12 @@ import { store } from 'store/index';
 import config from 'config';
 import App from 'containers/App';
 
+ReactGA.initialize('UA-147471357-1', {
```

Comment:
I think we should include both the `isProd` flag and this UA string in the `/configuration` endpoint. What do `debug` and `gaOptions/cookieDomain` do?